src/main/java/org/mule/extension/db/internal/result/resultset/NullResultSetHandler.java

```diff
@@ -30,4 +31,9 @@ public ResultSet processResultSet(DbConnection connection, ResultSet resultSet)
   public boolean requiresMultipleOpenedResults() {
     return false;
   }
+
+  @Override
+  public Charset getCharset() {
```

Comment:
Maybe is better to return the default charset?

Reply:
It shouldn't matter since this handler does not process anything.